source/includes/_cb.md

```diff
@@ -32,7 +32,7 @@
     "start_year": "Please enter starting year",
     "working_hours": "Your must provide Working Hours for at least one day in the week",
     "campaign_status": "Invalid campaign status",
-    "client_id": "Client with  ID 0 not found or doesn't belong to this customer"
+    "location-id": "Location with ID 0 not found or doesn't belong to this customer"
```

Comment:
Seems to be inconsistency. Is it locstion-id or location_id? The format should match the API in question.